### DISCUSSION TITLE
Question regarding Inheritance genotype filter for trios

### DISCUSSION THREAD

**Discussion by tommyli** [broadinstitute/seqr#3470] [Q&A]
Hi,

I'd like to ask about the expected behaviour when using the genotype inheritance filter.  Our most recent merge with upstream (as of 2023-07-05, previous merge was 2023-01-16) resulted in some behaviour change in results.  We noticed when only one sample has a genotype selected in a family of trio, then only that sample includes genotype information in the variant results.  Is this expected behaviour?  I noticed the genotype dropdown does not include the "any" option (i.e. 0, 1 and/or 2 alt alleles) but blank used to behave that way.  I'd like to understand the use case(s) for this behaviour.

See below screenshots:

![ScreenShot 2023-07-10 at 10 26 01 pm](https://github.com/broadinstitute/seqr/assets/117335/fd1a4690-68f2-47a2-aa11-aba342501671)

![ScreenShot 2023-07-10 at 10 31 33 pm](https://github.com/broadinstitute/seqr/assets/117335/b536e872-3668-478c-b9a6-f90c523a9ab0)

Sorry, I tried using taking screenshots from https://seqr.broadinstitute.org/ instead but I had trouble logging in (infinite OAuth redirects 🤷 ).

FYI, after some debugging, this is the line for this behaviour: https://github.com/broadinstitute/seqr/blob/c200fdc3eee153efde298027946099be92cde59c/seqr/utils/search/utils.py#L356

**Comment by hanars** [comment]
thanks for identifying this, it is a bug, all the available genotypes should be shown
